rust/agama-dbus-server/src/questions.rs

```diff
@@ -227,10 +239,35 @@ impl Questions {
         Ok(())
     }
 
-    /// sets questions to be answered by default answer instead of asking user
-    async fn use_default_answer(&mut self) -> Result<(), Error> {
-        log::info!("Answer questions with default option");
-        self.answer_strategies.push(Box::new(DefaultAnswers {}));
+    /// property that defines if questions is interactive or automatically answered with
+    /// default answer
+    #[dbus_interface(property)]
+    fn interactive(&self) -> bool {
+        let last = self.answer_strategies.last();
+        if let Some(real_strategy) = last {
+            real_strategy.id() == DefaultAnswers::id()
```

Comment:
What? Is this logic reversed?

As I understand it:
- *interactive* means that the UI pops up the question for the user to click on
- *non-interactive* means either
  - using the default answers for everything, or
  - reading the answers from a file (TODO)

Also an interesting point is whether the strategy is global (it is now, right?) or per question class (too complicated, not worth it)

Reply:
you are right. It should be `!=` ...and yes, it is global...if you want per question class you can achieve it by using answers file where you specify class and answer you want.